site/content/docs/5.3/components/close-button.md

```diff
@@ -47,6 +47,38 @@ Close buttons can also be created without `.btn-close` to reduce the size of you
   <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false"><use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"></use></svg>
   <span class="visually-hidden">Close</span>
 </button>
+
```

Comment:
As discussed with our tech lead, we can keep this part but we will need to add a callout at the beginning of the page to explain why we need the examples in this section in Boosted.